src/_data/sidenav.yml

```diff
@@ -34,7 +34,7 @@
 - title: Samples & tutorials
   children:
     - title: Flutter Gallery [updated]
-      permalink: https://flutter.github.io/gallery/
+      permalink: https://gallery.flutter.dev
```

Comment:
I don't have any context for this. Is it expected that you're replacing a link to code with a link to a running example? Those seem like very different use cases.